docs/hive/_index.md

```diff
@@ -65,7 +90,7 @@ For example, setting this in the `hive-site.xml` loaded by Spark will enable the
 Starting with Apache Iceberg `0.11.0`, when using Hive with Tez you also have to disable vectorization (`hive.vectorized.execution.enabled=false`).
 {{< /hint >}}
 
-#### Table property configuration
+##### Table property configuration
```

Comment:
Keep in mind these changes may impact what shows up in the table of contents for this page. IIRC, h5 headers don't show up in the TOC.

Reply:
I think this is fine.